src/main/java/friday/logic/parser/AddCommandParser.java

```diff
@@ -34,21 +34,22 @@ public class AddCommandParser implements Parser<AddCommand> {
     public AddCommand parse(String args) throws ParseException {
         ArgumentMultimap argMultimap =
                 ArgumentTokenizer.tokenize(args, PREFIX_NAME, PREFIX_TELEGRAMHANDLE, PREFIX_CONSULTATION,
-                        PREFIX_MASTERYCHECK, PREFIX_TAG);
+                        PREFIX_MASTERYCHECK);
 
-        if (!arePrefixesPresent(argMultimap, PREFIX_NAME, PREFIX_MASTERYCHECK, PREFIX_TELEGRAMHANDLE,
-                PREFIX_CONSULTATION)
+        if (!arePrefixesPresent(argMultimap, PREFIX_NAME)
                 || !argMultimap.getPreamble().isEmpty()) {
             throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddCommand.MESSAGE_USAGE));
         }
 
         Name name = ParserUtil.parseName(argMultimap.getValue(PREFIX_NAME).get());
-        TelegramHandle telegramHandle = ParserUtil.parsePhone(argMultimap.getValue(PREFIX_TELEGRAMHANDLE).get());
+        TelegramHandle telegramHandle = ParserUtil.parseTelegramHandle(argMultimap
+                .getValue(PREFIX_TELEGRAMHANDLE).orElse(""));
         Consultation consultation = ParserUtil.parseConsultation(LocalDate.parse(argMultimap
-                .getValue(PREFIX_CONSULTATION).get()));
+                .getValue(PREFIX_CONSULTATION).orElse("2001-01-01")));
         MasteryCheck masteryCheck = ParserUtil.parseMasteryCheck(LocalDate.parse(argMultimap
-                .getValue(PREFIX_MASTERYCHECK).get()));
+                .getValue(PREFIX_MASTERYCHECK).orElse("2001-01-01")));
```

Comment:
Might be good to abstract "2001-01-01" out to avoid magic string

Additionally, would it be possible to use null instead for no value? As by using "2001-01-01" it might prevent users from using this date.